pom.xml

```diff
@@ -34,6 +34,9 @@
     <version.jgit>4.11.0.201803080745-r</version.jgit>
 
     <version.cz.xtf>0.6</version.cz.xtf>
+    <version.org.keycloak>4.1.0.Final</version.org.keycloak>
```

Comment:
Lately there has been version update for Keycloak in XTF - https://github.com/xtf-cz/xtf/commit/b3bb4df582a8bacc976eee73a2e4d3be9d76ccb4 .
Is this definition needed or we can use updated keycloak from XTF?